test/regression/smart_answers_regression_test.rb

```diff
@@ -66,6 +66,10 @@ def teardown_hooks_installed?
         next if self.class.setup_has_run? && !self.class.teardown_hooks_installed?
         stub_content_api_default_artefact
         WebMock.stub_request(:get, WorkingDays::BANK_HOLIDAYS_URL).to_return(body: File.open(fixture_file('bank_holidays.json')))
+        WebMock.stub_request(:get, "#{Plek.new.find('imminence')}/areas/IG6%202BA.json").
+          to_return(body: File.open(fixture_file('imminence/london.json')))
+        WebMock.stub_request(:get, "#{Plek.new.find('imminence')}/areas/B1%201PW.json").
+          to_return(body: File.open(fixture_file('imminence/national.json')))
```

Comment:
I don't think you need these two lines. The `imminence_has_areas_for_postcode("B1%201PW",   [{ slug: "birmingham-city-council", country_name: 'England' }])` line below this seems to do what you need.